Gopkg.toml

```diff
@@ -33,6 +33,13 @@ required = [
   name = "k8s.io/code-generator"
   branch = "master"
 
+[[constraint]]
+  name = "k8s.io/apimachinery"
+  branch = "release-1.7"
```

Comment:
Note that this file misses all revision constraints for dependencies (which you only have in Gopkg.lock, which is ignored completely by dep when importing your new openshift/client-go into another project). This is even worse for transitive dependencies. There is no way at all to specify them in your Gopkg.toml for 3rdparty consumers (dep will just ignore them as they are no direct imports). This means that users cannot vendor this (without tons of manual overrides) at all. https://github.com/golang/dep/issues/1124